server/src-lib/Hasura/Eventing/ScheduledTrigger.hs

```diff
@@ -488,7 +497,7 @@ processScheduledEvent
           headers = addDefaultHeaders $ map encodeHeader sefHeaders
           extraLogCtx = ExtraLogContext (Just currentTime) sefId
           webhookReqPayload =
-            ScheduledEventWebhookPayload sefId sefName sefScheduledTime sefPayload sefComment currentTime
+            ScheduledEventWebhookPayload sefId sefName sefScheduledTime sefPayload sefComment sefCreatedAt
```

Comment:
I wonder if we should introduce a new payload field `invoked_at` for `currentTime`

Reply:
When I added `created_at` initially (before this PR), I was thinking about the invocation time, but clearly `created_at` is a bad name for it. I think there's value to add `invoked_at` in the payload.